sdk/keyvault/keyvault-certificates/test/utils/recorderUtils.ts

```diff
@@ -1,5 +1,12 @@
 import { env } from "@azure/test-utils-recorder";
 import { retry as realRetry } from "./retry";
+import { isNode } from "@azure/core-http";
+import * as dotenv from "dotenv";
+import * as path from "path";
+
+if (isNode) {
+  dotenv.config({ path: path.resolve(process.cwd(), "../.env") });
```

Comment:
Why this?
`"../.env"` should be enough?

Reply:
it depends who's calling it! Path resolution in node is kind of confusing to me, so I really prefer to have something predictable like this.

Reply:
I'll use __dirname